Sources/TuistKit/Commands/Cloud/CloudProjectListCommand.swift

```diff
@@ -0,0 +1,26 @@
+import ArgumentParser
+import Foundation
+import TSCBasic
+import TuistSupport
+
+struct CloudProjectListCommand: AsyncParsableCommand {
+    static var configuration: CommandConfiguration {
+        CommandConfiguration(
+            commandName: "list",
+            _superCommandName: "project",
+            abstract: "List projects you have access to."
+        )
+    }
+
+    @Option(
+        name: .long,
+        help: "URL to the cloud server."
+    )
+    var serverURL: String?
```

Comment:
I thought we'd be using `Config.swift` as a configuration surface for this. Does this override that value?

Reply:
`tuist cloud project list` works independently of your project (you should be able to run it from anywhere). Otherwise, yeah, `Config.swift` is used.